general-superstaq/general_superstaq/superstaq_client_test.py

```diff
@@ -783,12 +783,13 @@ def test_find_api_key() -> None:
         assert gss.superstaq_client.find_api_key() == "tomyheart"
 
     # find key in a config file
-    with mock.patch("pathlib.Path.is_file", return_value=True):
-        with mock.patch("builtins.open", mock.mock_open(read_data="tomyheart")):
-            assert gss.superstaq_client.find_api_key() == "tomyheart"
+    with mock.patch.dict(os.environ, clear=True):
```

Comment:
Actually, to fix the Windows pytest error something like this could work (to keep [other environment variables](https://github.com/SupertechLabs/client-superstaq/actions/runs/4539557022/jobs/7999541808?pr=419#step:5:83) set, e.g., `USERPROFILE`, `HOMEPATH`):
```suggestion
    mock_environ = os.environ.copy()
    del mock_environ["SUPERSTAQ_API_KEY"]
    with mock.patch.dict(os.environ, mock_environ):
```

Reply:
ah yep, didn't realize windows needed those! looks like just setting SUPERSTAQ_API_KEY to an empty string works

Reply:
Nice, yeah way cleaner.